File: CursoPOOUber/Java/Tarjeta.java
package Java;

import java.sql.Date;

class Tarjeta extends Payment{
    String banco;
    Date fechaVencimiento;
    Integer cvv;

    public Tarjeta(Integer id, String banco, Date fechaVencimiento, Integer cvv){
        super(id);
        this.banco = banco;
        this.fechaVencimiento = fechaVencimiento;
        this.cvv = cvv;

        printdataPayment();
    }
    
}

File: CursoPOOUber/Python/tarjeta.py
from datetime import date
from payment import Payment

class Tarjeta(Payment):
    banco               = str
    fechaVencimiento    = date.today()
    cvv                 = int 

    def __init__(self, id, fechaVencimiento, cvv):
        super().__init__(id)
        self.fechaVencimiento = fechaVencimiento
        self.cvv = cvv
        